extensions/amp-access/amp-access-spec.md

```diff
@@ -319,10 +320,20 @@ Notice the use of a URL hash parameter “status”. The value is either “true
 An integration with *amp-analytics* is under development and can be tracked on [Issue #1556][10]. This document will be updated when more details on the integration are available.
 
 #CORS Origin Security
```

Comment:
A few changes: Would you mind copying this into `ampproject/spec/amp-cors-requests.md` ?

Lets add back the automatic URL param. Otherwise they don't know what to put into the response header.